### DISCUSSION TITLE
to set sitemap

### DISCUSSION THREAD

**Discussion by Bamschool** [fuma-nama/fumadocs#525] [Q&A]
Hi! I really appreiciate the devs making Fumadocs. I really love it. However, I think if you share how to set simtemap.xml with it. I need to dynamically check for items that come after “docs” in the sitemap. I would really appreciate it if you could show me how to do this. Thank you!

**Comment by fuma-nama** [comment]
So you want to build a sitemap for the docs? You can use packages like next-sitemap or just use the Sitemap API of Next.js App Router (please refer to their docs).
